libs/langchain/langchain/chains/combine_documents/base.py

```diff
@@ -14,6 +18,21 @@
 from langchain.chains.base import Chain
 from langchain.text_splitter import RecursiveCharacterTextSplitter, TextSplitter
 
+LanguageModelLike = Union[
```

Comment:
this should probably go in core, https://github.com/langchain-ai/langchain/pull/15084/files#diff-639e6297e1db3729c228f5172ad900f6b5385c04ffef072fba2c4acd76de7e8d has it there